returnn/datasets/meta.py

```diff
@@ -1539,6 +1544,14 @@ def _collect_single_seq(self, seq_idx):
                     key,
                     sub_dataset_keys,
                 )
+            for key in self.pad_truncate_data_to_multiple_of:
+                assert (
+                    key in sub_dataset_keys
+                ), "%s: pad_truncate_data_to_multiple_of key %r not in sub dataset data-keys %r" % (
+                    self,
+                    key,
+                    sub_dataset_keys,
+                )
```

Comment:
Better use f-strings.
```suggestion
                assert key in sub_dataset_keys, (
                    f"{self}: pad_truncate_data_to_multiple_of key {key}"
                    f" not in sub dataset data-keys {sub_dataset_keys}"
                )
```